lib/agent0/agent0/interactive_fuzz/fuzz_long_short_maturity_values.py

```diff
@@ -49,7 +52,9 @@ def fuzz_long_short_maturity_values(num_trades: int):
     # TODO: set block time really high after contracts deployed:
     # chain_config = LocalChain.Config(block_time=1_000_000)
     # TODO: We will want to log the random seed; so remove this pylint disable once we do
-    chain, random_seed, rng, interactive_hyperdrive = setup_fuzz(log_filename)  # pylint: disable=unused-variable
+    chain, _, rng, interactive_hyperdrive = setup_fuzz(
```

Comment:
I left this unused variable here because we need to log it in a follow-up PR (today). The TODO above this line and the pylint disable on this line are also related to this.